setup.py

```diff
@@ -45,7 +45,7 @@ def run_tests(self):
         },
       cmdclass={'test': PyTest},
       platforms='any',
-      python_requires='>=3.5',
+      python_requires='>=3.6',
       classifiers=[
           'Programming Language :: Python',
           'Development Status :: 4 - Beta',
```

Comment:
@RemiLehe what do you think about changing that one as well? :)
```suggestion
          'Development Status :: 5 - Production/Stable',
```